src/linter/used_module_finder/index.coffee

```diff
@@ -1,27 +1,35 @@
-_ = require 'lodash'
-async = require 'async'
-asyncHandlers = require 'async-handlers'
-ExecutedModuleFinder = require './executed_module_finder'
-RequiredModuleFinder = require './required_module_finder'
-
-
 class UsedModuleFinder
 
-  constructor: ({@dir, ignoreFilePatterns}) ->
-    @executedModuleFinder = new ExecutedModuleFinder {@dir}
-    @requiredModuleFinder = new RequiredModuleFinder {@dir, ignoreFilePatterns}
+  constructor: ({@dir, @ignoreFilePatterns}) ->
 
 
   find: (done) =>
-    async.parallel [
-      (taskDone) => @requiredModuleFinder.find taskDone
-      (taskDone) => @executedModuleFinder.find taskDone
-    ], asyncHandlers.transform(@normalizeModules, done)
-
-
-  normalizeModules: (modules...) ->
+    async = require 'async'
+    asyncHandlers = require 'async-handlers'
+    handler = asyncHandlers.transform @normalizeModules, done
+    async.parallel {
+      executedModules: @getExecutedModules
+      requiredModules: @getRequiredModules
+    }, handler
+
+
+  # Private
+  getExecutedModules: (done) =>
```

Comment:
Using a leading underscore for private methods has the advantage that it becomes clear if a private method is accessed from the outside. It also makes clear who is private and who is not when debugging/inspecting objects. I think overall we had pretty good experiences with this pattern on ASF. How do you feel about adopting this pattern here?